.github/workflows/tools.yml

```diff
@@ -21,11 +21,9 @@ jobs:
             subsystem: tools
             label: tools
             run: |
-              NEW_VERSION=$(npm view eslint dist-tags.latest)
-              CURRENT_VERSION=$(node -p "require('./tools/node_modules/eslint/package.json').version")
-              if [ "$NEW_VERSION" != "$CURRENT_VERSION" ]; then
+              NEW_VERSION=$(./tools/dep_updaters/update-eslint.sh | tail -n1)
```

Comment:
I'm worried this would make debugging harder, as it would hide any output from the script. An alternative would be

```sh
./tools/dep_updaters/update-eslint.sh > output && tail -n1 output >> $GITHUB_ENV
cat output
```

Reply:
It's way more elegant 👏🏼